lib/fastlane/plugin/appcenter/actions/appcenter_upload_action.rb

```diff
@@ -596,7 +620,17 @@ def self.example_code
             destinations: "Alpha",
             destination_type: "store",
             release_notes: "this is a store release"
-          )'
+          )',
+          'appcenter_upload(
+            api_token: "...",
+            owner_name: "appcenter_owner",
+            app_name: "testing_google_play_app",
+            file: "./app.aab",
+            destinations: "*",
+            destination_excludes: "Collaborators",
+            destination_type: "store",
+            release_notes: "this is a store release"
```

Comment:
This example is wrong - it would be incorrect to have a destination as "*" for a store release

Reply:
I agree. I need to change the destination_type

Reply:
And also `.aab` is only valid for google play :)
The example Fastfile should also be modified to show off this feature.